identifiers/designators.go

```diff
@@ -54,43 +54,46 @@ const (
 
 // attributes
 const (
-	DesignatorsToken          = "designators"
-	AttributeCustomerGUID     = "customerGUID"
-	AttributeRegistryName     = "registryName"
-	AttributeRepository       = "repository"
-	AttributeTag              = "tag"
-	AttributeCluster          = "cluster"
-	AttributeClusterShortName = "clusterShortName"
-	AttributeNamespace        = "namespace"
-	AttributeKind             = "kind"
-	AttributeName             = "name"
-	AttributeContainerName    = "containerName"
-	AttributeApiVersion       = "apiVersion"
-	AttributeApiGroup         = "apiGroup"
-	AttributeWorkloadHash     = "workloadHash"
-	AttributeIsIncomplete     = "isIncomplete"
-	AttributeSensor           = "sensor"
-	AttributePath             = "path"
-	AttributeResourceID       = "resourceID"
-	AttributeContainerScanId  = "containerScanId"
-	AttributeSyncKind         = "syncKind"
-	AttributeSBOMToolVersion  = "sbomToolVersion"
-	AttributeSecurityRiskID   = "securityRiskID"
-	AttributeK8sResourceHash  = "k8sResourceHash"
-	AttributeType             = "type"
-	AttributeOwner            = "owner"
-	AttributeRelated          = "relatedObjects"
-	AttributeLayerHash        = "layerHash"
-	AttributeImageRepository  = "imageRepository"
-	AttributeResourceHash     = "resourceHash"
-	AttributeComponentVersion = "componentVersion"
-	AttributeComponent        = "component"
-	AttributeSeverityScore    = "severityScore"
-	AttributeSeverity         = "severity"
-	AttributeCVEID            = "cveID"
-	AttributeCVEName          = "cveName"
-	AttributeControlID        = "controlID"
-	AttributeBaseScore        = "baseScore"
+	DesignatorsToken              = "designators"
+	AttributeCustomerGUID         = "customerGUID"
+	AttributeRegistryName         = "registryName"
+	AttributeRepository           = "repository"
+	AttributeTag                  = "tag"
+	AttributeCluster              = "cluster"
+	AttributeClusterShortName     = "clusterShortName"
+	AttributeNamespace            = "namespace"
+	AttributeKind                 = "kind"
+	AttributeName                 = "name"
+	AttributeContainerName        = "containerName"
+	AttributeApiVersion           = "apiVersion"
+	AttributeApiGroup             = "apiGroup"
+	AttributeWorkloadHash         = "workloadHash"
+	AttributeIsIncomplete         = "isIncomplete"
+	AttributeSensor               = "sensor"
+	AttributePath                 = "path"
+	AttributeResourceID           = "resourceID"
+	AttributeContainerScanId      = "containerScanId"
+	AttributeSyncKind             = "syncKind"
+	AttributeSBOMToolVersion      = "sbomToolVersion"
+	AttributeSecurityRiskID       = "securityRiskID"
+	AttributeSecurityRiskCategory = "securityRiskCategory"
+	AttributeSecurityRiskName     = "securityRiskName"
+	AttributeSmartRemediation     = "smartRemediation"
```

Comment:
Why do you need smartRemediation
In config service we need only static stuff - if this can change during the life time of the ticket we do not save it